src/test/java/io/stargate/sgv2/jsonapi/api/v1/CountIntegrationTest.java

```diff
@@ -335,7 +351,7 @@ public void withExistFalseOperator() {
           .post(CollectionResource.BASE_PATH, namespaceName, collectionName)
           .then()
           .statusCode(200)
-          .body("status.count", is(4))
+          .body("status.count", is(5))
```

Comment:
Why did this change? Doesn't seem like a perf optimization should change the result here

Reply:
@vkarpov15  Have inserted one more document for testing `moreData` flag. Because of the additional record count increased by 1 for the provided filter condition in this test case.

Reply:
Makes sense, thanks for clarifying.